internal/v4/relations_test.go

```diff
@@ -855,3 +833,24 @@ func (s metaAnyResponseById) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s metaAnyResponseById) Less(i, j int) bool {
 	return s[i].Id.String() < s[j].Id.String()
 }
+
+// mustParseResolvedURL parses a resolved URL in string form, with
+// the optional promulgated revision preceding the entity URL
+// separated by a space.
+func mustParseResolvedURL(urlStr string) *router.ResolvedURL {
```

Comment:
probably. i will consider possibilities for a future PR. perhaps we could bless the GoString form of ResolvedURL and put the parsing code into router itself.